### DISCUSSION TITLE
Deleting Phantom wallet from browser deletes all the related data that comes with the wallet?

### DISCUSSION THREAD

**Discussion by malqubbaj** [phantom/docs#232] [Help & Support]
### Summary

I was using a test device that had my hot phantom wallet on Chrome, but later deleted it. Now using this device a atomic stealer malware has managed to affect the device, and this type of malware steal wallet data from the browser. 

So my question here is, are my wallets exposed even though they're deleted from the browser before the malware? 

### Steps to Reproduce

Delete the phantom wallet from browsers + get affected by malware

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @malqubbaj,

Please direct this request to Phantom support by going to Settings > Help & Support in your wallet.
